src/libs/LangChain.Core/Chains/StackableChains/Agents/Crew/CrewAgent.cs

```diff
@@ -223,6 +167,6 @@ protected override async Task<IChainValues> InternalCall(IChainValues values)
             }
         }
 
-        throw new InvalidOperationException($"Max actions exceeded({_maxActions})");
```

Comment:
Microsoft recommends not using general exceptions. This is where InvalidOperationException comes in.